src/components/Blog/Post/Markdown/index.tsx

```diff
@@ -1,17 +1,39 @@
 import React from 'react'
 import rehypeReact from 'rehype-react'
+import cn from 'classnames'
 
 import * as styles from './styles.module.css'
 
 interface IMarkdownProps {
   htmlAst: Node
 }
 
+const Admonition: React.FC<{
+  title?: string
+  type?: 'info' | 'tip' | 'warn'
+}> = ({ title, type = 'info', children }) => {
+  const genericTitles = {
+    info: 'Info',
+    tip: 'Tip',
+    warn: 'Warning'
```

Comment:
Right now we have three options:
* ⚠️ Warning
* ℹ️ Info
* 💡 Tip 

Do we need any other options?

Reply:
This is enough I think.

Reply:
> @julieg18, is it possible to make it configurable/customizable?

When you say make it configurable/customizable, do you mean
* adding more type options?
* being able to create a custom type with props?

If the first one, yes, we can add as many types as we'd like to have :). We can add them in this pr or at a later date. 

If the second, I can add some custom prop options like `emoji` and `color`. Example:
```
<admon title="New in 3.0" emoji="🔥"` and `color="orange"`>

text text text

</admon>
```

Reply:
I think some sane presets with more granular customization options would be the best of both worlds.

For example, `<admon type="note" icon="github">` would use whatever color notes use but replace the icon with a github icon. Also, we'll probably want to have fully custom icons since I'm guessing not everything we'll want there (like GitHub) will be an emoji.

Reply:
> For example, <admon type="note" icon="github"> would use whatever color notes use but replace the icon with a github icon. Also, we'll probably want to have fully custom icons since I'm guessing not everything we'll want there (like GitHub) will be an emoji.

Good ideas, Roger! I'll update the code, adding just an icon prop!

Reply:
I've added a icon prop, so you can now add a different icon. `<admon icon="exclamation" type="warn" title="Unique Title"> will create: 

<img width="723" alt="image" src="https://user-images.githubusercontent.com/43496356/159518207-c7473483-d502-4fb1-bf8d-eabc39f81f98.png">

Current icon options: 
    tip - 💡
    info - ℹ️
    warn - ⚠️
    fire - 🔥
    exclamation - ❗
    beetle - 🐞

Reply:
> Nice! Maybe "beetle" would be better as "bug", and while not necessary for a first iteration I'm sure we'll want "github" and "gitlab" icons at some point in the future.

Agreed! Do you mean updating the emoji to 🐛 or just renaming the emoji?

Reply:
I just mean changing the keyword to "bug", but if we already use the caterpillar as a bug indication and the ladybug as something else then I suppose we should keep "beetle" as it is and add the caterpillar under "bug". 

I can't find an example of either in the docs already so I'm not sure.

Reply:
I vote we use the slack/GitHub terms. "bug" for 🐛 and "lady_beetle" for 🐞